docs/user-guide/cli-using-daemon-mode.md

```diff
@@ -1,46 +1,65 @@
 
-**Table of Contents:**
-
 - [Feature overview](#feature-overview)
-- [Install the zowe native executable](#install-the-zowe-native-executable)
-- [Running zowe commands with the zowe executable](#running-zowe-commands-with-zowe-executable)
+- [Installation notes](#installation-notes)
+- [Running Zowe commands in daemon mode](#running-zowe-commands-in-daemon-mode)
 
 ## Feature overview
 
-Daemon Mode significantly improves the performance of Zowe CLI commands by running Zowe CLI as a persistent background process (daemon).
+Daemon mode significantly improves the performance of Zowe CLI commands by running Zowe CLI as a persistent background process (daemon). Running Zowe CLI as daemon lets Zowe absorb the one-time startup of Node.js modules, which resusts in significantly faster responses to Zowe commands.
 
-## Install the zowe native executable
+**Important:** We do not recommend using daemon mode in an environment where multiple users use the same system. For example, a shared Linux server.
 
-You can download and install the zowe native executable from Zowe Downloads (preferred) or from the Zowe Github repository. To install the zowe native executable from Zowe Downloads, see [Install Zowe CLI from a download](cli-install-cli-next.md#install-zowe-cli-from-a-download).
+## Installation notes
 
-To download and install the zowe native executable from the Zowe Github repository:
+The following notes apply to running Zowe in daemon mode:
 
-1. Navigate to the Zowe CLI GitHub [repository](https://github.com/zowe/zowe-cli)
-2. Under the **Release** section, click **Native Client Release**.
-   The Download page is displayed.
-3. Click the link to download the executable (.tgz file) for your operating system.
-4. Unzip the zowe .tgz file and place the zowe (or zowe.exe) file into a directory which occurs on your PATH earlier than the directory containing the NodeJS zowe command.
+- To enable daemon mode, ensure that you installed [the @next release of Zowe CLI](cli-install-cli-next.md).
+- When you are running Zowe on a Windows operating system in a virtual environment (for example, Windows Sandbox), you might receive an error message that indicates that a library named VCRUNTIME140.dll is missing. To correct the error, install Visual C++ Redistributable for Visual Studio 2015. For more information, see [Download Visual C++ Redistributable for Visual Studio 2015](https://www.microsoft.com/en-us/download/details.aspx?id=48145).
+- When you are running Zowe on a Mac operating system, the Mac executable might need an exception for an unidentified developer. For more information, see [Open a Mac app from an unidentified developer](https://support.apple.com/guide/mac-help/open-a-mac-app-from-an-unidentified-developer-mh40616/mac).
```

Comment:
This Note no longer applies after the `zowe daemon enable` command was introduced 😋